### PR TITLE
Quiz progress

### PR DESCRIPTION
**Pull request by Peter-White**

### CODE REVIEW

File: quiz.js [hunk 1]
```diff
@@ -26,6 +37,30 @@ if (square(2) !== 4 || cube(3) !== 27 || quad(4) !== 256) {
 
 /* your code starts here */
 
+function sum(numbers, twiceOrHalf) {
+    var count = 0;
+
+    if (twiceOrHalf === undefined) {
+        for (var i = 0; i < numbers.length; i++) {
```
Comment (MaxBlaushild) [inline quiz.js:44]: you can save yourself a lot of typing by performing this loop outside of the conditional

File: quiz.js [hunk 2]
```diff
@@ -26,6 +37,30 @@ if (square(2) !== 4 || cube(3) !== 27 || quad(4) !== 256) {
...
+            count += numbers[i];
+        }
+    }
+    else if (twiceOrHalf === false) {
+        for (var k = 0; k < numbers.length; k++) {
+            count += numbers[k];
+        }
+        count / 2;
+    }
+    else {
```
Comment (MaxBlaushild) [inline quiz.js:54]: what happens when twiceOrHalf is 'truthy or falsey?

**Comment by MaxBlaushild** [comment]
Score: 3/5

This stuff is tricky, but just keep on plugging away!

**Comment by Peter-White** [comment]
I'll look at it tonight, I think I know how to get it working.

**Pull request closed without merging**